modules/search.xqm

```diff
@@ -47,13 +47,13 @@ declare variable $search:SECTIONS := map {
         map {
             "id": "visits",
             "query": function($query as xs:string) {
-                collection($config:VISITS_COL)//visits[ft:query(., $query, $search:ft-query-options)]
+                collection($config:VISITS_COL)//visits[ft:query(., 'content:' || $query, $search:ft-query-options)]
```

Comment:
To account for keyword searches containing more than one word, the query string should be wrapped in parentheses. E.g., from hsg-facets: `'hsg-fulltext:(' || $keywords || ')'`.